README.md

```diff
@@ -194,6 +195,13 @@ lsr_role2collection.py [-h] [--namespace NAMESPACE] [--collection COLLECTION]
                      in the README that you do not want to change, you may have to change the
                      README in another way, not using this script, by using sed with a custom
                      regex.
+--collection-mapping If it is set and given namespace.collection is not default, the default
+                     collection fedora.linux_system_roles is converted to
+                     NAMESPACE.COLLECTION.
```

Comment:
I thought we could already do this with `--extra-mapping`?

Reply:
No, `--extra-mapping` unfortunately does not cover this case for now...

Maybe, I'd better improve `parse_extra_mapping` to handle it, instead...

Reply:
ok - if `--extra-mapping` doesn't work, then add `--collection-mapping`

Reply:
Thanks, @richm. I've modified the parser a bit to support it.